### ISSUE TITLE
postinstall script incorrectly detects local appium install

### ISSUE THREAD

**Issue by boneskull** [appium/appium#17054]
I just noticed this when running from an empty temp dir:

```
❯ npm i -g appium@2.0.0-beta.40
(##################) ⠹ reify:es-abstract: timing reifyNode:node_
> appium@2.0.0-beta.40 postinstall
> node ./scripts/autoinstall-extensions.js

[Appium] ⠋ Checking Appium installation...(##################) ⠦
[Appium] ✔ Appium installation OK
[Appium] Found local Appium installation; skipping automatic installation of extensions.
```

That's not right.

**Comment by jlipps** [comment]
Yes, I think I ran afoul of this too, but didn't have the benefit of stderr output for some reason. I had tried to use `--drivers=xxx` and nothing happened.

**Comment by boneskull** [comment]
resolved by 965752f
